multi-node/vagrant/kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+clusters:
+- cluster:
+    certificate-authority: ssl/ca.pem
+    server: https://172.17.4.101:443
+  name: vagrant-cluster
+contexts:
+- context:
+    cluster: vagrant-cluster
+    namespace: default
+    user: vagrant-admin
+  name: vagrant-multi
+users:
+- name: vagrant-admin
```

Comment:
Even though this is a static config and it wouldn't exactly matter like it would with modifying existing user config, can we namespace these like the local-user examples too: `vagrant-multi-admin`